libnavigator/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+sdk=28
```

Comment:
@Guardiola31337 can tell this is on purpose. but is there an option to put this in one place?

libtesting-utils or libtesting-ui? or could we have the number in one place?

Reply:
can robolectric.properties see dependencies.gradle?

Reply:
Those are great questions, do you know? 😅